templates/bluemix/manifest.yml

```diff
@@ -0,0 +1,16 @@
+declared-services:
+  cloudant-db:
+    label: cloudantNoSQLDB
+    plan: Lite
+applications:
+- path: .
+  memory: 256M
```

Comment:
@raymondfeng @hacksparrow for not I'd just hard-code memory the a reasonable value. A user can always modify it manually afterwards.